src/System.Private.CoreLib/src/System/Threading/Tasks/TaskExceptionHolder.cs

```diff
@@ -59,28 +50,6 @@ internal TaskExceptionHolder(Task task)
         {
             Debug.Assert(task != null, "Expected a non-null task.");
             m_task = task;
-            EnsureADUnloadCallbackRegistered();
-        }
-
-        private static bool ShouldFailFastOnUnobservedException()
```

Comment:
Noticed this while replying to https://gitter.im/dotnet/coreclr?at=5b3a2ec4f16644066112c39d